setup/gcp/installer/forseti_installer.py

```diff
@@ -152,14 +178,14 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             if self.config.dry_run:
                 print('This is a dry run, will not copy any files.')
 
-            utils.print_banner('Copying configuration file/deployment'
-                               ' template to GCS')
+            utils.print_banner('Backing up important files to GCS')
```

Comment:
nit: CamelCase, fix throughout for banners.

Reply:
Updated.